libs/ha/ha_common_libs_k8s.py

```diff
@@ -778,17 +793,23 @@ def start_random_mpu(self, event, s3_data, bucket_name, object_name, file_size,
         res = (exp_failed_parts, failed_parts, parts_etag, mpu_id)
         output.put(res)
 
-    def check_cluster_status(self, pod_obj, pod_list=None):
+    def check_cluster_status(self, pod_obj, pod_list=None, dir_path=None):
         """
         :param pod_obj: Object for master node
         :param pod_list: Data pod name list to get the hctl status
+        :param dir_path : Path to repo scripts
         :return: boolean, response
         """
         LOGGER.info("Check the overall K8s cluster status.")
+        resp = None
         try:
-            resp = pod_obj.execute_cmd(common_cmd.CLSTR_STATUS_CMD.format(self.dir_path))
+            if dir_path:
```

Comment:
cmd_path = dir_path if dir_path else self.dir_path

The full if block can be reduced to a single line.